students/shibin_mathew/lesson6/test_mailroom.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+from collections import defaultdict
+from mailroom import *
+
+
+
+class MailroomTestCase(unittest.TestCase):
+    # Tests for 'mailroom.py'
+    def test_switch_menu(self):
+        self.assertEqual(switch_menu('1'), send_thank_you)
+        self.assertEqual(switch_menu('2'), create_report)
+        self.assertEqual(switch_menu('3'), send_letters)
+        self.assertEqual(switch_menu('4'), quit_menu)
+
+    def test_send_letters(self):
+        list_test = []
+        list_test2 = []
+        #  Test 1
+        for names in donors2:
+            list_test.append(names+'.txt')
+        self.assertEqual(send_letters(donors2), list_test)
+        #  Test 2
+        for names in donors:
+            list_test2.append(names+'.txt')
+        self.assertEqual(send_letters(donors), list_test2)
```

Comment:
This test would be more useful if it confirmed that the files were actually written to disk with the expected content